index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+      integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <div>
+      <nav>
+        <h3>My logo</h3>
+        <i class="fa-solid fa-bars"></i>
+      </nav>
+    </div>
+    <div class="landing">
+      <h1>Hey There.</h1>
+      <h1>I'm Alex.</h1>
```

Comment:
- [ ] For SEO reasons, it is not advisable for you to use two `<h1>` tags in this manner. To optimize your page for SEO and provide a clear structure, it is generally recommended to have only one `<h1>` tag per page, representing the main topic or title of the content.
- [ ] Additionally, please customize this section with your information.